### PR TITLE
Add Empty States

### PR DESCRIPTION
**Pull request by sigmachirality**
Screenshot from 2020-01-21 14-10-00
Screenshot from 2020-01-21 14-10-06
Screenshot from 2020-01-21 14-10-12
Screenshot from 2020-01-21 13-14-51

### CODE REVIEW

File: frontend/components/ClubPage/Tabs.js [hunk 1]
```diff
@@ -11,16 +12,16 @@ export default props => (
             <div
               style={{ whiteSpace: 'pre-wrap' }}
               dangerouslySetInnerHTML={{
-                __html: props.club.description || EMPTY_DESCRIPTION,
+                __html: club.description || EMPTY_DESCRIPTION,
```
Comment (cacabo) [inline frontend/components/ClubPage/Tabs.js]: Can you destructure props from the club object before rendering them?

File: frontend/components/ClubPage/Tabs.js [hunk 2]
```diff
@@ -1,7 +1,8 @@
 import { EMPTY_DESCRIPTION } from '../../utils'
+import { EmptyState } from '../common'
 import TabView from '../TabView'
 
-export default props => (
+export default ({ club }) => (
```
Comment (cacabo) [inline frontend/components/ClubPage/Tabs.js]:
```suggestion
export default ({ club: {
  thingsWithinClub
}}) => (
```

Comment (cacabo) [reply frontend/components/ClubPage/Tabs.js]: Would prefer destructuring

File: frontend/components/Settings/ClubTab.js [hunk 3]
```diff
@@ -66,55 +62,54 @@ export default props => {
...
+          </tr>
+        ))}
       </tbody>
     </Table>
+  ) : (
+      <>
+        <EmptyState name="button" />
+        <p className="has-text-grey" style={{ textAlign: 'center' }}>
+          You are not a member of any clubs yet.
+        </p>
```
Comment (cacabo) [inline frontend/components/Settings/ClubTab.js]: Maybe refactor this to use the `<Text .. />` component? Could add a `isGray` and `isCentered` props to it if it doesn't already have that functionality baked in.

File: frontend/components/Settings/FavoritesTab.js [hunk 4]
```diff
@@ -10,6 +11,7 @@ export default ({ clubs = [], favorites, keyword, updateFavorites }) => {
     return ret
   })
   const rows = Object.keys(table)
+  const isBookmarks = keyword === 'bookmark'
```
Comment (cacabo) [inline frontend/components/Settings/FavoritesTab.js]: Wouldn't correct english grammar be `areBookmarks`? 🤙

File: frontend/public/static/img/bookmarks.svg [hunk 5]
```diff
@@ -0,0 +1 @@
...6TmtQQkW1uotjqftXD0OVMEa1VHmaLB1MUXvzEiqquTHqiqWeQl2iYkXNl6WuP6yiOufOwu7rJS1VniZLQQ2eTFmtpoCtsVyTpd+Xeg66DurZzlb5urN6GM4iSxU1eIxq9W5Uq+NTgEE1xdqlt02WHkf5gtbAelAtVZ1Nln5vp5xznedS66Bafb2kpQnHGhG5A1sPLXZW1Tqop7saZcza9TebLFXVzlNU1oNq+SpNwNZkseigWppw+FjaHmbXb5ilh/w6d6esB5UVNR9FV1Trr2S0FNTOZ6csb6HeN7x9WrMU1KCu3+I6qqW/xDbFjVEtBtXSX2IbS8tTx5xzz7R9WAf1sbYGGSuholoKKtBRVS3v9bOi5qd15r/rnDvW9mHmLF7ls8nSFirQMfPfBXCq7cPMWbu81+cygEdao4x8qe2DXQDqLRWZMh9UEdmHrWfCOoN6vO3DzJkP6oqlo4zPt32wC7u7UqUE1dI2cecY1epk6rrWwAhLXX/n8pTFxX7A5r2oPpYq6nHnnHeplJOp/FkaowItB6g5mcqftSdtvduou7C5KwUwqLnyPuO/C5v7/AAnU7nyXpjGnan8Wauo3utPrXb9+wUcmq5ZO5jivYhiF8Djvg8yV0o1hYjcAnBHa5eR1q7fYlBLODnVZKmqtgbV4oJ/KROpmqWgeu/qZVBtsDTzbw2qxa7f2mvPNZYeu3nK95NWt1BL2eevWTrp31pRLa6j3tMaGGNpv/+I72CK1WN+t7QGxlgKKuDZRrV6KOWG1sAYS10/4HnlpNUx6gOtgTGWZv2A56jfLmy+CM3STk0Ia13/1qK/1XXU0oJqrev3jlEtKm3Wby2o3jGqRUXN+kXkIWyFtZiKel9rYJCloG49jmI1qKWcRW2yNPP3TqYsKrGiWjrp712esqiYg9MNlirqSefc2tb+Lmwu5ZQYVGtrqWsTql3YOnRbK+ES303Wgrq2RLULe79BwNYMOJS13/NWUC2NbWoWf08aawXn6eY/7AJ4s6Vhrq6ISImzfmtBXTtAvQvg9ZaGuXpPa2CUta5/K6g/hK0J1dtaA6M+AbCvNcrIelBXFxj8YUvjHP2X1sAiEXkEW2PzE81/2AUAEfk+gD/3Ns+PtaFMDEvj1LXnpg52pkTkTwD8JoC/Q74L5lcB/KvWyLAPtQa5Otr8BxF5DcBrzrkdVK9SeQ7VMsE
```
Comment (cacabo) [inline frontend/public/static/img/bookmarks.svg]: Are there not feather icon alternatives? Seems like these icons are bigger than they should be but if this is all we can get it'll work ok.

File: frontend/components/Settings/FavoritesTab.js (hunk 4 above)
Comment (sigmachirality) [reply frontend/components/Settings/FavoritesTab.js]: I've been writing too much code and not enough prose LOL

File: frontend/public/static/img/bookmarks.svg (hunk 5 above)
Comment (sigmachirality) [reply frontend/public/static/img/bookmarks.svg]: This is an empty state svg, I got it from the design team, so check in with them?

Comment (ezwang) [reply frontend/public/static/img/bookmarks.svg]: @cacabo, it looks like these svgs are not actually icons, but images. This seems fine to me!

**Pull request merged**